lib/omnibus/packagers/windows_base.rb

```diff
@@ -102,41 +107,41 @@ def timestamp_servers
       signing_identity[:timestamp_servers]
     end
 
+    def keypair_alias
+      signing_identity[:keypair_alias]
+    end
+
     def machine_store?
       signing_identity[:machine_store]
     end
 
-    #
-    # Iterates through available timestamp servers and tries to sign
-    # the file with with each server, stopping after the first to succeed.
-    # If none succeed, an exception is raised.
-    #
+    # signs the package with the given certificate
     def sign_package(package_file)
-      success = false
-      timestamp_servers.each do |ts|
-        success = try_sign(package_file, ts)
-        break if success
-      end
-      raise FailedToSignWindowsPackage.new unless success
+      raise FailedToSignWindowsPackage.new unless is_signed?(package_file)
     end
 
-    def try_sign(package_file, url)
+    def is_signed?(package_file)
       cmd = [].tap do |arr|
-        arr << "signtool.exe"
-        arr << "sign /v"
-        arr << "/t #{url}"
-        arr << "/fd #{algorithm}"
-        arr << "/sm" if machine_store?
-        arr << "/s #{cert_store_name}"
-        arr << "/sha1 #{thumbprint}"
-        arr << "/d #{project.package_name}"
-        arr << "\"#{package_file}\""
+        arr << "smctl.exe"
```

Comment:
What impacts are expected to client configuration by not supporting local cert store?

Reply:
Do you mean customer clients, or chef products? The reason we had to remove it from the machine store is because smctl installs the ssl as the running user, and in this case is the created windows user `buildkite-agent`. I am unaware of any impacts on the moving of the cert for this, as the only time I see the old thumbprint listed is when we sign the msi package. 

https://github.com/search?q=org%3Achef%2013B510D1CF1B3467856A064F1BEA12D0884D2528&type=code

https://github.com/search?q=org%3Ainspec%2013B510D1CF1B3467856A064F1BEA12D0884D2528&type=code